cmd/utils/flags.go

```diff
@@ -1242,6 +1259,38 @@ func SetNodeConfig(ctx *cli.Context, cfg *node.Config) {
 	}
 }
 
+func unmarshalBlockNumber(input string) rpc.BlockNumber {
```

Comment:
Can we just use [`BlockNumber.UnmarshalJSON`](https://github.com/scroll-tech/go-ethereum/blob/develop/rpc/types.go#L74)? And using a default is dangerous, let's just return the parse error to the user.

Reply:
umarshaljson doesn't support usual numbers, only hex

Reply:
TODO:
- Raise error on invalid hex/number passed
- Figure out what the default value is if user does not specify